source/common/stats/symbol_table_impl.cc

```diff
@@ -451,8 +451,8 @@ StatNameStorage::StatNameStorage(absl::string_view name, SymbolTable& table)
     : StatNameStorageBase(table.encode(name)) {}
 
 StatNameStorage::StatNameStorage(StatName src, SymbolTable& table) {
-  const uint64_t size = src.size();
-  MemBlockBuilder<uint8_t> storage(size);
+  const size_t size = src.size();
+  MemBlockBuilder<uint8_t> storage(size); // Note: MemBlockBuilder takes uint64_t.
```

Comment:
In theory `MemBlockBuilder` could be updated to use `size_t`, too, and I suspect there may be other places we could switch over as well. My feeling is that that's a bit outside the scope of this change, however.

Reply:
agreed; outside the scope for now.